website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Releasing Docusaurus i18n
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, i18n]
+image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release **[Docusaurus 2 i18n](https://v2.docusaurus.io/docs/i18n/introduction)**, as part of [2.0.0-alpha.71](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.71).
+
+**We went the extra mile**, and the new i18n support is **even better than in Docusaurus 1**.
+
+Docusaurus 2 has now reached **full feature parity with Docusaurus 1**, and a first **2.0 beta** is on its way.
+
+In this post, we will present you the **translation workflow**, explain some **design decisions** and **showcase early adopter sites**: [Jest](https://jestjs.io/), [Redwood](https://learn.redwoodjs.com/), and [Datagit](https://datagit.ir/).
+
+## Translate your site
+
+You can translate a Docusaurus site in **3 simple steps**:
+
+- **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
+- **Translate**: put the translation files at the correct file-system location.
+- **Deploy**: build and deploy your site using a single or multi-domain strategy.
```

Comment:
If you need to follow these steps in order, mabe use a ordered list instead of a unordered list.
You can use `1.`'s to create a ordered list.

Reply:
thanks, fixed